lib/screens/main_menu/menu_button.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class MenuButton extends StatelessWidget {
+  final Function onPressed;
+  final String buttonText;
+
+  MenuButton(this.buttonText, {Function onPressed}):
+    this.onPressed = onPressed; 
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(10.0),
+      child: RaisedButton(
+        onPressed: onPressed,
+        color: Colors.green,
```

Comment:
Could experiment with the theme a bit for this - from memory there are lots of things you can specify. For `Pocket Scat`, I think specifying `primaryColor` was enough to get my floating action button to automatically be purple.